### PR TITLE
Bump patchutils to 0.3.4

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/text/patchutils/test/results-64.master [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+PASS: tests/convert1/run-test
+PASS: tests/convert2/run-test
+PASS: tests/clean1/run-test
+PASS: tests/clean2/run-test
+PASS: tests/stdin/run-test
+PASS: tests/splitdiffD/run-test
+PASS: tests/fullheader1/run-test
+PASS: tests/fullheader2/run-test
+# TOTAL: 137
+# PASS:  132
```
Comment (pyhalov) [inline components/text/patchutils/test/results-64.master:139]: \# PASS:  133, \# FAIL: 0 here...

Comment (alarcher) [reply components/text/patchutils/test/results-64.master:139]: erf... lucky you, not sure what differs...

Comment (pyhalov) [reply components/text/patchutils/test/results-64.master:139]: Did you do 'gmake test' or 'gmake install test'?

Comment (alarcher) [reply components/text/patchutils/test/results-64.master:139]: I just tried both, same result.

Comment (pyhalov) [reply components/text/patchutils/test/results-64.master:139]: What do you see in tests/lsdiff2/run-test.log ?

Comment (alarcher) [reply components/text/patchutils/test/results-64.master:139]:
```
[9] mkdir dir dir.orig
[10] echo a
[11] echo A
[12] gdiff -uN dir.orig dir
[13] rm dir/a
[14] gdiff -uN dir.orig dir
[15] mv dir.orig/a dir
[16] gdiff -uN dir.orig dir
[17] cat diff diff
[18] /scratch/alarcher/oi-userland/components/text/patchutils/build/amd64/src/filterdiff --list -s --strip=1 diff2
[19] '[' -s errors ']'
[21] cat
[21] cmp - index
- index differ: char 5, line 2
[21] exit 1
FAIL tests/lsdiff2/run-test (exit status: 1)
```

Comment (pyhalov) [reply components/text/patchutils/test/results-64.master:139]: Strange, but I can't longer reproduce this, see the declared test results...

**Pull request merged**